ci/tests/harness.h

```diff
@@ -3,7 +3,7 @@
  * It is included by every benchmark file and provides the time measuring,
  * as well as standardized command-line argument parsing.
  *
- *  Release under Public domain.
+ *  Released under Public domain.
  *  Author:  S. Marr  University of Cambridge
```

Comment:
Does this mean Stefan was at U Cambridge? I don't think he ever was?

Reply:
[89c86f9](https://github.com/capablevms/bdwgc/pull/18/commits/89c86f9a5d2e84155909d89269561b7359026a37)